### PR TITLE
Typescriptification

### LINKED ISSUE #3: Typescript would be nice (but maybe not necessary)

### PR DESCRIPTION
**Pull request by dannyreilman**
Straight dabbing on the haters this thing is ts now

### CODE REVIEW

**Comment by dannyreilman** [comment]
We can refactor this I just wanted it working

**Review by ZachSanders21 — APPROVED** [review]
Lets Go!

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,11 @@
...
 node_modules/
 
+# built js
+dist/
+
 # secret token
+src/config.json
+
+# leaving this in case people still have the file here but it needs to be in src/
 config.json
```
Comment (BDokas) [inline .gitignore:5]: I think we should remove it just so we can catch people slipping and hold their bots hostage

File: README.md [hunk 2]
```diff
@@ -10,13 +10,14 @@ Discord bot for shooting_the_breeze.java
...
 2. [Adding your bot to servers](https://discordjs.guide/preparations/adding-your-bot-to-servers.html)
-3. Setup a config.json as shown (you'll need to enable dev settings to get the client and guild id through the right-click in discord):
+3. Setup a config.json as shown in the src/ folder (you'll need to enable dev settings to get the client and guild id through the right-click in discord):
 ```
 {
 	"clientId": "your-bots-id",
 	"guildId": "your-guild-id",
 	"token": "your-token-goes-here"
 }
 ```
```
Comment (BDokas) [inline README.md:20]: Is there an install step that isn't captured in the readme?

Comment (dannyreilman) [reply README.md:20]: Yeah but I'm too lazy to do that in this pr I'll make a separate issue

**Review by BDokas — APPROVED** [review]

**Pull request merged**